cdc/sink/mysql.go

```diff
@@ -106,6 +106,25 @@ type mysqlSink struct {
 }
 
 func (s *mysqlSink) EmitRowChangedEvents(ctx context.Context, rows ...*model.RowChangedEvent) error {
+	if len(rows) > 0 {
+		resolvedTs := rows[0].CommitTs - 1
+		for {
+			oldResolvedTs := atomic.LoadUint64(&s.resolvedTs)
+			if oldResolvedTs <= resolvedTs || atomic.CompareAndSwapUint64(&s.resolvedTs, oldResolvedTs, resolvedTs) {
+				break
+			}
```

Comment:
After this change, the upper caller of sink must have a level trigger mechanism (current processor/sink manager has  ensured), would you please add some comments here about it, in case that we change the behavior by accident in the future.